packages/server/graphql/queries/helpers/connectionFromGitHubIssues.ts

```diff
@@ -0,0 +1,29 @@
+import {GetIssuesNodeFragment} from '../../../types/typed-document-nodes'
+
+interface GitHubConnIssue extends GetIssuesNodeFragment {
+  updatedAt: Date
+}
+
+const connectionFromGitHubIssues = <T extends {updatedAt: Date} = GitHubConnIssue>(
```

Comment:
i think we can get rid of this whole thing, we should just return the connection exactly as github gives it to us